### PR TITLE
changed time calculation

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob <jakob.roehrl@web.de>

* Resolves: # 
* Target version: vue

### CODE REVIEW

**Comment by juliushaertl** [comment]
At some point we should also make sure the computed property is update every minute or so, since otherwise keeping the page open without any update will lead to outdated relative timestamps.

**Comment by juliushaertl** [comment]
@jakobroehrl Mind to look into this as well? :)

**Comment by jakobroehrl** [comment]
@juliushaertl: Yes, sure. Good idea. 
How can I retrigger the computed property?

File: src/components/card/CardSidebar.vue [hunk 1]
```diff
@@ -133,21 +138,30 @@ export default {
...
 	watch: {
-		'currentCard': {
-			immediate: true,
-			handler() {
-				this.copiedCard = JSON.parse(JSON.stringify(this.currentCard))
-				this.allLabels = this.currentCard.labels
-				this.assignedUsers = this.currentCard.assignedUsers.map((item) => item.participant)
-				this.desc = this.currentCard.description
-			}
+		currentCard() {
```
Comment (juliushaertl) [inline src/components/card/CardSidebar.vue]: Any reason you removed the immediate property of the watcher? Iirc that was needed to properly initialize the data on initial load.

Comment (jakobroehrl) [reply src/components/card/CardSidebar.vue]: readded

**Pull request merged**